lib/services/local_storage/local_storage_service.dart

```diff
@@ -10,6 +10,11 @@ abstract class LocalStorageService {
 
   Future<T?> getObject<T extends Mappable>(String key);
 
+  Future<void> setListObject<T extends Mappable>(List<T> list,
+      {required String key});
+
+  Future<List<T>?> getListObject<T extends Mappable>(String key);
```

Comment:
imo, you shouldn't store the data when it's a complex data rather than primitive type (string, bool...) as a json. The issue with this approach could be that later when you have a structure changed, you release a new version, it could potentially NOT recognize your stored json anymore. You can temporarily do this for now, but I'd recommend you to look at some other nosql or sqlite alternative.

Reply:
@sleepylee for sure, I just try to keep it simple for this IC only 👍

Reply:
ok for now